API-GUIDELINE.md

```diff
@@ -0,0 +1,254 @@
+# API GUIDELINES
+
+## Overview
+As commercetools is 100% API First, it's important to have a well designed developer experience.
+Additionally we aim to achieve more automation and improve the review process of new changes in the APIs.
+
+With this in mind, we have created "API Guidelines" that aims to help developers to design APIs:
+
+ - simple
+ - consistent 
+ - self-explanatory
+ 
+The Keywords to have a good Guidelines are:
+ 
+ - Consistency => following the Guidelines
+ - Uniformity => checking in the code if there are other properties with similar name and adjust in base of it or improve it
```

Comment:
It is not clear what "adjust(ing) in base of it or improv(ing) it" means. "In base of" presumably means "on the basis of" or "based on"? What do you adjust? Perhaps adjust(ing) "field naming" based on it? The final phrase "or improve it" rather implies that there are other ways to improve the code, if not, maybe we can omit it. I suggest the following:

```suggestion
 - Uniformity => checking in the code if there are other properties with similar names and adjusting field naming based on it
```